schemas/test/en/test_timeout.json

```diff
@@ -4,7 +4,7 @@
     "schema_version": "0.0.1",
     "data_version": "0.0.3",
     "survey_id": "0",
-    "session_timeout_in_seconds": 5,
+    "session_timeout_in_seconds": 70,
```

Comment:
Changed to 5, added mention about non JS test, removed the spec file, it will be tested now as part of `schemas/test/en/test_timeout_modal.json` now, thanks!